pymc4/random_variables/discrete.py

```diff
@@ -17,7 +20,7 @@ class Constant(RandomVariable):
     _base_dist = tfd.Deterministic
 
 
-''' TODO DiscreteUniform raises a NotImplementedError from tfp.
+# TODO DiscreteUniform raises a NotImplementedError from tfp.
```

Comment:
This comment should be removed now right?

Reply:
ah ya. duh.